ts/sagas/messages.ts

```diff
@@ -161,9 +176,43 @@ export function* loadMessages(): Iterator<Effect> {
 
       yield put(loadMessagesSuccess());
     }
+  } finally {
+    if (yield cancelled()) {
+      // If the task is cancelled send a failure message
```

Comment:
not sure if this is semantically correct, it's not an error (that is usually an unforeseen/unhandled state) if the task was canceled on purpose